packages/one-app-runner/src/startApp.js

```diff
@@ -59,38 +43,39 @@ async function startAppContainer({
     [
       'run',
       '-t',
-      ...ports.map((port) => `-p=${port}:${port}`),
-      ...[...envVars.entries()].map(([envVarName, envVarValue]) => `-e=${envVarName}=${envVarValue}`),
-      ...[...mounts.entries()].map(([hostPath, containerPath]) => `-v=${hostPath}:${containerPath}`),
-      name ? `--name=${name}` : null,
-      network ? `--network=${network}` : null,
+      portsToDockerArgs(ports),
+      envVarsToDockerArgs(envVars),
+      mountsToDockerArgs(mounts),
+      objectToFlags({
+        name,
+        network,
+      }),
       imageReference,
       '/bin/sh',
       '-c',
       containerShellCommand,
-    ].filter(Boolean),
+    ].flat(),
```

Comment:
are we fine with falsy values being passed now? is spawnAndPipe just handling that?

Reply:
Falsy values are no longer added in cases where one of the args is not being set. Closest would be one of those functions generating an empty array, which will be removed by flat()